From 453ddb755f34355b4c45e253416e59c02d15fe70 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin2.jansa@lgepartner.com>
Date: Fri, 18 Aug 2023 00:12:55 +0200
Subject: [PATCH] CMakeLists.txt: respect libdir setting

:Release Notes:

:Detailed Notes:
Fixes:
http://gecko.lge.com:8000/Errors/Details/665168

ERROR: QA Issue: lib32-com.webos.service.mediaindexer: Files/directories were installed but not shipped in any package:
  /usr/lib/libmediaindexerclient.so
Please set FILES such that these items are packaged. Alternatively if they are unneeded, avoid installing them or delete them within do_install.
lib32-com.webos.service.mediaindexer: 1 installed and not shipped files. [installed-vs-shipped]

when libdir is /usr/lib32 or /usr/lib64.

:Testing Performed:
Only build tested.

:QA Notes:
No change to image.

:Issues Addressed:
[WRP-20118] Bitbake world issues 2023-07
[WRP-24156] CCC: Various build fixes

Change-Id: I92a08e166b92d8a34afa575704b77a6b88714cce
---
Upstream-Status: Submitted [http://gpro.lge.com/c/webosose/com.webos.service.mediaindexer/+/365247 CMakeLists.txt: respect libdir setting]

 src/mediaindexerclient/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mediaindexerclient/CMakeLists.txt b/src/mediaindexerclient/CMakeLists.txt
index 5630fd7..507e8ec 100644
--- a/src/mediaindexerclient/CMakeLists.txt
+++ b/src/mediaindexerclient/CMakeLists.txt
@@ -72,5 +72,5 @@ target_link_libraries(mediaindexerclient
     ${PMLOG_LDFLAGS}
     )
 
-install(TARGETS mediaindexerclient DESTINATION lib)
+install(TARGETS mediaindexerclient DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(FILES ${MEDIA_INDEXER_CLIENT_HEADERS} DESTINATION include/mediaindexerclinet)
